pages/pfpcollections.tsx

```
@@ -36,6 +37,13 @@ const PFPCollections: NextPage = () => {
             ))}
           </div>
         </section>
+      {/* </div> */}
+      <div className={styles.buttonContainer}>
+        <Button onClick={()=>{}}>
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)